mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/impl/ClientState.scala

```diff
@@ -273,8 +278,6 @@ import scala.util.{Failure, Success}
     timer =>
       if (data.keepAlive.toMillis > 0)
         timer.startSingleTimer("send-pingreq", SendPingReqTimeout, data.keepAlive)
-      else
-        data.remote.complete() // We'll never be sending pings so free up the command channel for other things
```

Comment:
This was a hangover from an older approach, and is no longer valid - there can now be many uses of the `data.remote` `Source`, beyond `PINGREQ`.